### PR TITLE
SortByLength.rb

### LINKED ISSUE #8: Sort by Length
Create a function that takes a string and returns a string ordered by the length of the words. From shortest to the longest word. If there are words with the same amount of letters, order them alphabetically.

## Examples
```
sort_by_length("Hello my friend") ➞ "my Hello friend"

sort_by_length("Have a wonderful day") ➞ "a day Have wonderful"

sort_by_length("My son loves pineapples") ➞ "My son loves pineapples"
```

## Notes
Punctuation (periods, commas, etc) belongs to the word in front of it.

### PR DESCRIPTION
**Pull request by Anishluke92**

### CODE REVIEW

File: SortByLength.rb [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+Punctuation (periods, commas, etc) belongs to the word in front of it.
+
+Algorithm 
+Lets take a sentence, for example "Hello my dear world". And in that sentence we see that it consist of four words.
+Let take the first word and count its alphabets to get the word size.
+then lets do the same to rest of the words in that sentence.
+Now we got to know all the word size in that sentence. Lets rearrange the sentence by sorting the words by size in ascending order. example,
+"my dear hello world"
+=end
+def sort_by_length(strings)
```
Comment (danielpaul) [inline SortByLength.rb]: spacing

https://github.com/rubocop-hq/ruby-style-guide#empty-lines-around-bodies

File: SortByLength.rb [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+    strings.split(" ").sort_by(&:length).join(" ")
```
Comment (danielpaul) [inline SortByLength.rb]: can you do this without `.sort_by`? Use one of the sorting algorithms you learnt to sort it manually.

**Review by danielpaul — CHANGES_REQUESTED** [review]

File: SortByLength.rb (hunk 2 above)
Comment (Anishluke92) [reply SortByLength.rb]: I can do it

Comment (Anishluke92) [reply SortByLength.rb]: I have done it !

**Pull request merged**